CHANGELOG.md

```diff
@@ -1,19 +1,21 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* Support mirroring of binary files. Thanks to @malice00. ([#2501](https://github.com/realm/realm-js/issues/2501))
+* Performance significantly improved when making a query on the property of a linked table, when the property is indexed. ([]())
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None.
+* Fixed an out-of-range bug in Windows builds. In debug mode, the exception `can't dereference out of range vector iterator` will be thrown and the process will terminate. ([realm/realm-object-store#832](https://github.com/realm/realm-object-store/issues/832))
+* A race between extending the file and activity in the encryption layer could lead to crash and corruption. This race has been fixed. The bug may hit on Android, if encryption is in use. It could also affect Linux on file systems where POSIX `prealloc()` is unsupported. ([realm/realm-core#3427](https://github.com/realm/realm-core/pulls/3427))
```

Comment:
```suggestion
* A race between extending the file and activity in the encryption layer could lead to crash and corruption. This race has been fixed. The bug could hit on Android, if encryption was in use. It could also affect Linux on file systems where POSIX `prealloc()` was unsupported. ([realm/realm-core#3427](https://github.com/realm/realm-core/pulls/3427))
```